src/main/resources/com/google/api/codegen/py/docs/conf.py.snip

```diff
@@ -317,5 +317,4 @@
   napoleon_use_ivar = False
   napoleon_use_param = True
   napoleon_use_rtype = True
-
```

Comment:
Right now, the rendered templates end in _two_ newlines, which is why I removed this (so they would only end in one).